runtime/src/accounts_db.rs

```diff
@@ -3339,21 +3319,20 @@ impl AccountsDb {
 
         let slot_list = lock.slot_list();
         let (slot, info) = slot_list[index];
-        let store_id = info.store_id();
-        let offset = info.offset();
+        let location = info.storage_location();
```

Comment:
Would you be ok with `storage_location` for the var instead of `location`? That would help me at least 😇

(And same for the other spots with `location`)